pkg/cloud/gcp/actuators/machine/actuator.go

```diff
@@ -41,59 +57,154 @@ func NewActuator(params ActuatorParams) *Actuator {
 // Create creates a machine and is invoked by the machine controller.
 func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machine *machinev1.Machine) error {
 	klog.Infof("Creating machine %q", machine.Name)
-	scope, err := newMachineScope(machineScopeParams{
-		machineClient: a.machineClient,
-		coreClient:    a.coreClient,
-		machine:       machine,
-	})
+
+	machineCtx, err := newMachineContext(a.coreClient, machine)
 	if err != nil {
-		return fmt.Errorf(scopeFailFmt, machine.Name, err)
+		return err
+	}
+
+	instance, err := instanceGet(machineCtx, machine.Name)
```

Comment:
This might be redundant as we call Exists() in the controller before we call Create().